README.md

```diff
@@ -23,3 +23,9 @@ The project can be build by using `make`. Along with the usual `make`, `make cle
 Tests are located in `demo/` and can be built in two flavors.
 * `make test` creates a test binary that is intended to be run against `mtest`. `mtest` can be built with `make mtest` and test execution is done like `./mtest/mtest | ./test`. `mtest` is creating test vectors using an alternative MPI library and `test` is consuming these vectors to verify correct behavior of ltm
 * `make test_standalone` creates a stand-alone test binary that executes several test routines.
+
+[Valgrind](http://valgrind.org/) does not support `long double` but does not reset the limits in `float.h` accordingly. Please build `demo.c` with the additional compiler flag `-DLTM_MEMCHECK_VALGRIND`, e.g.:
```

Comment:
What does that mean? To which instructions translate the long double operations? Are they not present in the valgrind cpu? On x86 or x86_64?

Reply:
Valgrind makes quite a mess with `long double`. Uses `double` instead but doesn't tell anybody, well, can't tell anybody.

Reply:
One more argument to just get rid of long double support here.